invoices/resolution_result.go

```diff
@@ -142,7 +142,7 @@ func (f FailResolutionResult) FailureString() string {
 		return "invoice already canceled"
 
 	case ResultInvoiceAlreadySettled:
-		return "invoice alread settled"
+		return "invoice already settled"
```

Comment:
This might be used in error messages. And since it's invoice related, it might be possible this change breaks someone's app where the developer string matched the error message to detect if an invoice was already paid...

Reply:
It's clearly a typo but I'd still be cautious here. What do you think, @bhandras?

Reply:
Missed this one, I think we shouldn't change it as it's an error message.

Reply:
I strongly disagree but I reverted it. Deliberately keeping an error because of fear that someone may depend on this error is just wrong mentality.